### ISSUE TITLE
Question about other login methods and features

### ISSUE THREAD

**Issue by arianmaykon** [zeropingheroes/lanager#74]
Question, i know the whole project is based on Steam, but i see some cases where the LAN Party actually happened without internet, due to several reasons. I know LAN parties are not supposed to happen without internet, but it's a possibility. In this case, the Lanager could run online, so before the LAN Party the people could be invited, invite, propose, vote, organize, etc. And at the LAN Party, if no internet is available, at least the instructions could be saved locally (more on this below).

What do you think about introduce other login systems (e-mail, Facebook, etc) on Lanager? It's possible or should be another project? I mean, if this would not break the projects principle to you guys/community.

I'm willing to help with the issues on my spare time. But i just want to know about this once this would be the main reason to me to contribute/use the project, so it can be used by my friends and community.

If possible, other features should be implemented to a better fit with this idea, IMHO.
Some of them are (and i think some are already implemented/suggested):
- Invite friends to a LAN party (through e-mail/Facebook/etc);
- Allow users to propose and vote in games;
- Allow users to attach media to a proposed game, screenshots, videos, etc, so the others can see how the game is before vote for it;
- Allow the creation of pages/instructions or something like this, to help users to share games/instructions;

I know it is a long text, and maybe i was not very clear about it, but let me know if you need anything to understand or discuss about this.

Thanks.

**Comment by ilumos** [comment]
Hi Arian, thanks for taking an interest in the project. I'll try to answer your questions one by one:

**Decoupling from Steam & providing other auth options**
*Short answer:* In the distant future
*Long answer:* I am trying to build the system in a way that will allow for this without too much difficulty, but arguably Steam is the de facto standard for PC gaming right now and for the forseeable future. It also has a very good API that adds a lot of value to the system, so it is not a priority at all right now.

**Removing requirement for internet access**
*Short answer:* In the distant future
*Long answer:* Due to the Steam integration this is not currently possible. Once the above is done we can make steps toward the above. The system has very small bandwidth requirements, so could work on a poor internet connection with a bit of tuning. Also as with the rest of the project, we're trying to cater for the majority of LAN parties, and the majority have internet access of some kind.

**Invite friends to a LAN party**
Pre-event functionality will be tackled in a separate app, designed to be hosted on the internet. It will take care of promotion, tickets, seating etc, and integrate with the LANager via APIs where necessary

**Allow users to propose games**
Planned - see issue #71 

**Allow users to attach media to a proposed game**
Added to #71 

**Allow the creation of pages**
This is currently implemented, but only admins can create pages. I would say that any instructions / media should be part of a game proposal as above

I hope this helps - let me know

**Comment by arianmaykon** [comment]
Yes, it helped a lot, thanks. I agree with you about Steam and all, but lots of thing can affect how some users and groups/communities can be balanced in a way all can play together, hard to explain.

Thanks

**Comment by ilumos** [comment]
Yes I know different groups have preferred services and my idea of "the standard" is based on my limited experience. The plan is definitely to stop being so dependant on Steam, as who knows what the future holds, but for now I am concentrating on other features that will hopefully add more value.

Thanks again :)
